### PR TITLE
[Fixes: #17027] Fix spacing between own messages

### PR DESCRIPTION
**Pull request by cammellos**
status: ready

### CODE REVIEW

**Comment by cammellos** [comment]
@Francesca-G could you please have a look at the spacing between your own messages? It should be fixed in this PR.
Thank you!

**Review by flexsurfer — APPROVED** [review]

File: src/status_im2/contexts/chat/messages/content/view.cljs [hunk 1]
```diff
@@ -134,9 +134,14 @@
...
-                                :padding       8
-                                :opacity       (if (and outgoing (= outgoing-status :sending)) 0.5 1)}
+          :style               {:border-radius      16
+                                :padding-horizontal 8
+                                :padding-vertical   (if (:first-in-group? message-data)
+                                                      8
+                                                      4)
+                                :opacity            (if (and outgoing (= outgoing-status :sending))
+                                                      0.5
+                                                      1)}
```
Comment (ajayesivan) [inline src/status_im2/contexts/chat/messages/content/view.cljs]: Considering we have multiple lines of style code, would it be more organized to relocate this to the dedicated style file? wdyt

**Comment by erikseppanen** [comment]
Is the last message supposed to have a slight gap above it? I mean, the messages are not expected to all be equidistant from each other? If that's true, then LGTM

**Review by erikseppanen — APPROVED** [review]

**Review by Francesca-G — APPROVED** [review]
Looks good ✨

**Pull request merged**